exercise/sitemaps.py

```diff
@@ -18,12 +18,25 @@ def items(self):
             audience=LearningObject.AUDIENCE.COURSE_AUDIENCE,
```

Comment:
`BaseExerciseSitemap` should cover independent exercises that are not part of any chapter. Embedded exercises in chapters are not listed separately because their parent chapter pages are listed in `CourseChapterSitemap`.